docs/wiki/Upgrading.md

```diff
@@ -1,24 +1,51 @@
-# Upgrade Process
+# Updating Eventum versions
```

Comment:
may be: Upgrade steps or Upgrade guidelines

Reply:
this is page heading. there's such subsection already: "Upgrade Process", altho there's other section that imho should be renamed: "Upgrading to latest version"

Reply:
May be just "Upgrading Eventum"?